src/core/Akka.Streams/Dsl/Source.cs

```diff
@@ -194,7 +194,7 @@ public Source(IModule module)
                     switch (reply)
                     {
                         case TOut2 a: return a;
-                        case Status.Success s when s.Status is TOut2 a: return a;
```

Comment:
The rest of the changes in this file is mostly C# syntax modernization or adding `ActorSystem` overloads.